crates/tauri/src/webview/webview_window.rs

```diff
@@ -2057,6 +2057,33 @@ impl<R: Runtime> WebviewWindow<R> {
     self.window.set_fullscreen(fullscreen)
   }
 
+  /// Toggles a fullscreen mode that doesn't require a new macOS space.
+  ///
+  /// This is how fullscreen used to work on macOS in versions before Lion.
+  /// It allows the user to have a fullscreen window without using another space
+  /// or taking control over the entire monitor.
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **macOS:** Uses native simple fullscreen mode.
+  /// - **Other platforms:** Falls back to [`Self::set_fullscreen`].
+  #[cfg(target_os = "macos")]
+  pub fn set_simple_fullscreen(&self, fullscreen: bool) -> crate::Result<()> {
+    self.window.set_simple_fullscreen(fullscreen)
```

Comment:
Since we handle this in `set_simple_fullscreen` anyways, I think we just use a single function here that does the forwarding